### DISCUSSION TITLE
AttributeError: module 'xarray' has no attribute 'open_dataset'

### DISCUSSION THREAD

**Discussion by zaratia** [pydata/xarray#5525] [Q&A] [answered]
Hi, when I call the open_dataset attribute from xarray I get this error message:

AttributeError: module 'xarray' has no attribute 'open_dataset'

I do not understand what happened, it was regularly woking...

**Comment by keewis** [comment, marked as answer]
this usually is caused by a broken environment. Could you check whether there's a file called `xarray.py` (or a directory called `xarray` containing python files) in the current directory or in any of the paths listed in `sys.path`? If that's not helping you would have to try setting up a new environment (using `conda` or `virtualenv`).

Edit: if all of that does not help we would need more information to be able to help
